requirements-dev.txt

```diff
@@ -1,16 +1,13 @@
-setuptools==44.0.0
```

Comment:
The latest version is required **Py3.7**  https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v5970